components/ratgdo/ratgdo.cpp

```diff
@@ -44,25 +44,6 @@ namespace ratgdo {
 
     void RATGDOComponent::setup()
     {
-        this->rolling_code_counter_pref_ = global_preferences->make_preference<int>(734874333U);
-        uint32_t rolling_code_counter = 0;
-        this->rolling_code_counter_pref_.load(&rolling_code_counter);
-        this->rolling_code_counter = rolling_code_counter;
-        // observers are subscribed in the setup() of children defer notify until after setup()
-        defer([=] { this->rolling_code_counter.notify(); });
```

Comment:
We should try to load the old value and remove it in a few months so we don't have a breaking change here

Reply:
The esp8266 design is not so great. But nothing we can do about that.

The other option would be to do a one time change the remote id so it starts a new counter, but I'm not sure thats better